### PR TITLE
Fixes 419

### PR DESCRIPTION
**Pull request by prisis**

### CODE REVIEW

**Comment by staabm** [comment]
in case we start to support symfony/var-dumper < 3.x this should be reflected in the composer.json

**Comment by prisis** [comment]
What is the best way to do it?

Should i do something like ^2.6 || ^3.0 ?

**Comment by staabm** [comment]
I guess so.

**Comment by prisis** [comment]
Added a new matrix for php 5.6 for lowest install

File: .travis.yml [hunk 1]
```diff
@@ -9,12 +9,22 @@ php:
...
+      env: SETUP=lowest
+
 cache:
   directories:
     - "$HOME/.composer/cache"
 
 before_script:
-  - composer install --no-interaction --prefer-dist
+  - if [[ "$SETUP" = "basic" ]]; then composer install --no-interaction --prefer-dist; fi
+  - if [[ "$SETUP" = "lowest" ]]; then travis_retry composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable; fi
```
Comment (staabm) [inline .travis.yml]: why the travis_retry in this particular case?

File: src/Whoops/Util/TemplateHelper.php [hunk 2]
```diff
@@ -119,8 +119,15 @@ public function dump($value)
         if ($dumper) {
             // re-use the same DumpOutput instance, so it won't re-render the global styles/scripts on each dump.
             // exclude verbose information (e.g. exception stack traces)
+            if (class_exists('Symfony\Component\VarDumper\Caster\Caster')) {
+                $cloner = $this->getCloner()->cloneVar($value, Caster::EXCLUDE_VERBOSE);
```
Comment (staabm) [inline src/Whoops/Util/TemplateHelper.php]: this var should be renamed to `clone`

**Comment by staabm** [comment]
thanks looks mostly good!

File: src/Whoops/Util/TemplateHelper.php (hunk 2 above)
Comment (prisis) [inline src/Whoops/Util/TemplateHelper.php]: done :)

**Comment by prisis** [comment]
@filp any info for this pr?

**Comment by prisis** [comment]
@filp any news for this?

**Comment by denis-sokolov** [comment]
Thank you for your contribution, this seems good.

**Pull request merged**